module/buff/basic_buff.cpp

```diff
@@ -574,9 +578,10 @@ void Detector::findTarget(cv::Mat&                              _input_dst_img,
   fmt::print("[{}] Number of fans: {}\n", idntifier_green, _target_box.size());
 }
 
-bool Detector::isFindTarget(cv::Mat& _input_img, std::vector<abstract_target::Target>& _target_box) {
+bool Detector::isFindTarget(cv::Mat&                              _input_img,
+                            std::vector<abstract_target::Target>& _target_box) {
   if (_target_box.size() < 1) {
-    fmt::print("[{}] Info, no target detected\n", idntifier_green);
+    fmt::print("[{}] Info, XXX no target detected XXX \n", idntifier_green);
```

Comment:
```suggestion
    fmt::print("[{}] Info, no target detected \n", idntifier_green);
```
建议加入 idntifier_yellow